internal/hardware/validator.go

```diff
@@ -515,9 +515,14 @@ func (v *validator) getOCPRequirementsForVersion(openshiftVersion string) (*mode
 func (v *validator) getValidDeviceStorageTypes(hostArchitecture string, openshiftVersion string) []string {
 	validTypes := []string{string(models.DriveTypeHDD), string(models.DriveTypeSSD), string(models.DriveTypeMultipath)}
 
-	isGreater, err := common.BaseVersionGreaterOrEqual("4.15.0", openshiftVersion)
+	isGreater, err := common.BaseVersionGreaterOrEqual("4.14.0", openshiftVersion)
 	if err == nil && isGreater {
-		validTypes = append(validTypes, string(models.DriveTypeISCSI))
+		validTypes = append(validTypes, string(models.DriveTypeRAID))
+
+		isGreater, err = common.BaseVersionGreaterOrEqual("4.15.0", openshiftVersion)
+		if err == nil && isGreater {
+			validTypes = append(validTypes, string(models.DriveTypeISCSI))
+		}
```

Comment:
Please move this out of the 4.14 "if" statement.  I get the optimization, but it makes the code harder to read and maintain.